clients/mobile/src/utility/formatte.ts

```diff
@@ -0,0 +1,43 @@
+export const formatteDate = (date: string | Date): string => {
+  const dte = new Date(date)
+  const formattedDate = dte.toLocaleString(undefined, {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    hour: 'numeric',
+    minute: '2-digit',
+    hour12: true
+  })
+  return formattedDate
+}
+
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  phoneNumber = phoneNumber.trim()
```

Comment:
are you changing `inputs?` please don't; this is a risky habit

Reply:
Yes vai, We are giving options like to enter phone numbers with country codes and without country codes.

Reply:
we can't be changing the inputs of a function. That goes against a lot of functional programming principles.

Reply:
Vai, is it okay to first copy the phone number modify that, and return it?

```
export const formatPhoneNumber = (phoneNumber: string): string => {
  const trimmedPhoneNumber = phoneNumber.trim();
  
  return trimmedPhoneNumber.startsWith('01')
    ? trimmedPhoneNumber.replace('01', '+8801')
    : trimmedPhoneNumber;
}

```

Reply:
yes, this is better